<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheatManagerWindow</class>
 <widget class="QWidget" name="CheatManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cheat Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/duck.png</normaloff>:/icons/duck.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cheatListNewCategory">
       <property name="text">
        <string>&amp;Add Group...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cheatListAdd">
       <property name="text">
        <string>&amp;Add Code...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cheatListEdit">
       <property name="text">
        <string>&amp;Edit Code...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cheatListRemove">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Delete Code</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cheatListActivate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Activate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cheatListImport">
       <property name="text">
        <string>Import...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cheatListExport">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Export...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cheatListClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cheatListReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTreeWidget" name="cheatList">
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Activation</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Instructions</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
